index.js

```diff
@@ -9,7 +9,7 @@ import App from './src/App';
 import { resources } from './src/languages';
 
 // Languages (i18n)
-lang.defaultLocale = 'en';
+lang.defaultLocale = 'fr';
```

Comment:
hey @marco ! was this for testing purposes?

Reply:
I think that was originally from Skylar's branch, but yes I'd imagine debugging, I will undo that change!